sql/api/src/main/scala/org/apache/spark/sql/errors/ExecutionErrors.scala

```diff
@@ -53,6 +53,12 @@ private[sql] trait ExecutionErrors extends DataTypeErrorsBase {
       e)
   }
 
+  def stateStoreHandleNotInitialized(): SparkRuntimeException = {
```

Comment:
Can we move this to the `StateErrors.scala` to keep error classes in this area in a common location ?

Reply:
@ericm-db Could you move it, please.

Reply:
@MaxGekk we can't move it to `StateStoreErrors.scala` because we can't use it in `sql/api/` where we throw the error

Reply:
I see. ok.